tensorflow/lite/micro/memory_planner/greedy_memory_planner.cc

```diff
@@ -315,13 +334,14 @@ void GreedyMemoryPlanner::PrintMemoryPlan(ErrorReporter* error_reporter) {
   CalculateOffsetsIfNeeded();
 
   for (int i = 0; i < buffer_count_; ++i) {
+#ifndef TF_LITE_STRIP_ERROR_STRINGS
+    char s[] = {GetOrdinalCharacter(i), 0};
     TF_LITE_REPORT_ERROR(
         error_reporter,
-        "Planner buffer ID: %d, calculated offset: %d, size required: %d, "
-        "first_time_created: %d, "
-        "last_time_used: %d",
-        i, buffer_offsets_[i], requirements_[i].size,
+        "%s (id=%d): size=%d, offset=%d, first_used=%d last_used=%d", s, i,
+        requirements_[i].size, buffer_offsets_[i],
```

Comment:
we are incrementally moving away from TF_LITE_ERROR_REPORTER in favor of using MicroPrintf:
https://github.com/tensorflow/tensorflow/blob/282fa073abc720a3b668c0aebe607c4105c3effe/tensorflow/lite/micro/micro_error_reporter.h#L26

Can you switch this over to
```cc
MicroPrintf("%s (id=%d): size=%d, offset=%d, first_used=%d last_used=%d", s, i, requirements_[i].size, buffer_offsets_[i],requirements_[i].first_time_used, requirements_[i].last_time_used);
```